samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -625,6 +624,13 @@
                 }
               ]
             }
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "dynamodb:ListStreams"
+            ],
+            "Resource": "*"
```

Comment:
Could we tighten the resources that this applies to? This way, they can list all the streams for the table that they specify in the policy template, or all streams if the user specifies a "*" for TableName.

```json
            "Resource": {
              "Fn::Sub": [
                "arn:${AWS::Partition}:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${tableName}/stream/*",
                {
                  "tableName": {
                    "Ref": "TableName"
                  }
                }
              ]
            }
```